src/esp/gfx_batch/Renderer.cpp

```diff
@@ -240,89 +291,131 @@ void Renderer::addFile(const Cr::Containers::StringView filename,
   //  for glb, bps and ply?)
   CORRADE_INTERNAL_ASSERT_OUTPUT(importer->openFile(filename));
```

Comment:
Yeah, not great at all, and I have to fix it as it asserts on me quite often :D Should be something more graceful.

I'm thinking returning a `bool` here and then asserting / `ESP_CHECK()`ing when you forget to check the failure and attempt to add a non-existent name (from that failed-to-load file, for example).

Reply:
Done, the function now returns a `bool` and all non-programmer errors (i.e., caused by corrupted files or unexpected data) are now runtime failures instead of assertions

Note that I'm not using Habitat's logging APIs in order to avoid linking to `core`, based on [our discussion in the previous PR](https://github.com/facebookresearch/habitat-sim/pull/1798#discussion_r911398937).